quartz_solar_forecast/data.py

```diff
@@ -163,22 +163,33 @@ def make_pv_data(site: PVSite, ts: pd.Timestamp) -> xr.Dataset:
             raise ValueError("Multiple sites found matching the given latitude and longitude.")
         else:
             site_id = matching_site_ids[0]
-            live_generation_wh = get_solaredge_data(site_id)
+            live_generation_kw = get_solaredge_data(site_id)
     elif site.inverter_type == 'enphase':
-        live_generation_wh = get_enphase_data(system_id)
+        live_generation_kw = get_enphase_data(system_id)
+
+        if live_generation_kw is not None:
+            # get the most recent data
+            recent_pv_data = live_generation_kw[live_generation_kw['timestamp'] <= ts]
+            power_kw = np.array([np.array(recent_pv_data["power_kw"].values, dtype=np.float64)])
+            timestamp = recent_pv_data['timestamp'].values
+            
+            lon = [site.longitude] * len(power_kw)
+            lat = [site.latitude] * len(power_kw)
+            pv_id = [1] * len(power_kw)
+        else:
+            # make fake pv data, this is where we could add history of a pv system
+            power_kw = [[np.nan]]
+            timestamp = [ts]
```

Comment:
When you're in this else bit when `live_generation_kw` is None do you need to specify lat and long again?

Reply:
> When you're in this else bit when `live_generation_kw` is None do you need to specify lat and long again?

From the inverter_example code i guess yes, since I'm comparing it with when we're using enphase VS when we're not, so when I remove this bit, it asks for the credentials again and goes into the else block of the enphase condition, which is kinda not needed, so to ease things off I've added it again

Reply:
> When you're in this else bit when `live_generation_kw` is None do you need to specify lat and long again?

This seems right, I've removed the redundant bits of code